internal/validation/extension.go

```diff
@@ -122,14 +123,28 @@ func (s *Server) Process(srv pb.ExternalProcessor_ProcessServer) error {
 				}
 				err = s.validate(req, service, operation)
 				if err != nil {
+					errMsg := ErrorBody{}
```

Comment:
I suspect this is intentional but why not use and define `NewErrorBody` then continue to use `errMsg.Error`?

Reply:
Thanks, i got sidetracked with something else